client/starwhale/utils/config.py

```diff
@@ -248,3 +257,42 @@ def update_instance(
         )
 
         update_swcli_config(**self._config)
+
+
+@click.group(
+    "config",
+    cls=AliasedGroup,
+    help="Configuration management, edit is supported now",
+)
+def config_cmd() -> None:
+    pass
+
+
+@config_cmd.command("edit", aliases=["e"], help="edit the configuration of swlci")
+def __edit() -> None:
+    _edit()
+
+
+def _edit() -> None:
+    _editor = ""
+    if platform == "linux" or platform == "linux2":
```

Comment:
use `sys.platform` is better than `platform` global variable?

Reply:
platform related code is removed